### PR TITLE
Deployment configuration

### PR DESCRIPTION
**Pull request by markrawls**

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -8,3 +8,4 @@ djangorestframework-simplejwt==4.4.0
 cryptography==2.8
 simplejwt-extensions==0.1.0
 django-anymail==7.0.0
+zappa==0.51.0
```
Comment (cviveiro) [inline backend/requirements.txt:11]: Should this be standard and managed by zygoat?

File: backend/zappa_settings.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "base": {
+    "aws_region": "us-east-1",
+    "django_settings": "backend.settings",
+    "profile_name": "default",
+    "project_name": "portunus-backend",
+    "runtime": "python3.7",
+    "s3_bucket": "com.willing.portunus-deployments"
```
Comment (cviveiro) [inline backend/zappa_settings.json:8]: Missing comma

File: backend/requirements.txt (hunk 1 above)
Comment (cviveiro) [reply backend/requirements.txt:11]: Oh no, that's right, we took deployment out. Nevermind.

Comment (markrawls) [reply backend/requirements.txt:11]: I am looking at adding this into our plugin repository however, but that's outside the scope of this PR.

File: backend/zappa_settings.json (hunk 2 above)
Comment (markrawls) [reply backend/zappa_settings.json]: Ran prettier against the file to make it a little more consistent.

**Pull request merged**